doc/connectivity/networking/native_sim_setup.rst

```diff
@@ -142,3 +142,18 @@ For TCP test, type:
 .. code-block:: console
 
    ./echo-client -t 127.0.0.1
+
+Setting interface name from command line
+****************************************
+
+By default the ethernet interface name used by native_sim is determined by
+:kconfig:option:`CONFIG_ETH_NATIVE_POSIX_DRV_NAME`. There is option to choose
+the interface on command line.
+
+First you have to enable :kconfig:option:`CONFIG_ETH_NATIVE_POSIX_DRV_NAME_FROM_CMD_LINE`.
+
+Then to select the interface you have to pass ``--eth-if`` command line option.
```

Comment:
Could you add here info about the use case when user would want to use this option. You have this mentioned in the commit message but it would be nice to see it here too.